src/main/java/com/github/dockerjava/api/model/PushResponseItem.java

```diff
@@ -18,7 +18,7 @@
      */
     @JsonIgnore
     public boolean isErrorIndicated() {
-        if (getError() == null)
+        if (getError() == null && getErrorDetail() == null)
```

Comment:
If either is not null, its an error.  Only when both are null (i.e. 'and') is it safe to return false.